home_work_3/seminar_task_6.py

```diff
@@ -0,0 +1,15 @@
+# Задание №6 семинара 3. Работа с выводом строки.
+
+my_str = input('Введите строку: ')
+my_str = my_str.split()
+my_str = sorted(my_str)
+len_max = 0
+counter = 1
+
+for item in my_str:
+    if len(item) > len_max:
+        len_max = len(item)
```

Comment:
Значение переменной `len_max` можно найти проще, используя встроенные агрегирующие функции.
```suggestion
len_max = len(max(my_str, key=len))
```